KappaAnalysis/src/KappaEnumTypes.cc

```diff
@@ -9,7 +9,8 @@ KappaEnumTypes::GenParticleType KappaEnumTypes::ToGenParticleType(std::string co
 	else if (genParticleName == "genElectron") return KappaEnumTypes::GenParticleType::GENELECTRON;
 	else if (genParticleName == "genMuon") return KappaEnumTypes::GenParticleType::GENMUON;
 	else if (genParticleName == "genTau") return KappaEnumTypes::GenParticleType::GENTAU;
-	else return KappaEnumTypes::GenParticleType::NONE;
+	LOG(FATAL) << "Unkown genParticleType: " << genParticleName;
```

Comment:
Will this ever be reached?

Reply:
Sorry, the return after the log fatal is needed, because it won't compile if it's not there. The complier will give an error about a missing return value, since it doesn't know that a log fatal terminates the program.